### DISCUSSION TITLE
Exact usage of Colmap data

### DISCUSSION THREAD

**Discussion by l-fungus-l** [pierotofy/OpenSplat#122] [General]
I am currently working to transform a dataset from a software that Opensplat does not support into a Colmap dataset that I can then use. From analyzing the code it seems to me that only the points3D.bin file is utilized for the initialization of pSet. However in the [format](https://colmap.github.io/format.html) of points3D there are references to the other files such as where a specific point is found in an image, as well as its position in the 2D point-list of each image (the Track[] arguements that are much harder to add and I hope to skip). Is this information used in any way? As long as I have ID, X, Y, Z, R, G, B values for points will Opensplat effectively be able to reconstruct a scene?

**Comment by pierotofy** [comment]
You should only need camera information and points. OpenSplat doesn't need track information. So yes?

**Reply by l-fungus-l** [reply]
Does camera information include the xy of the tiepoint on the specific image?

**Reply by pierotofy** [reply]
Nop.
